src/main/app/src/Components/CommandComponent/Command.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import Terminal from 'react-console-emulator';
+import CommandContext from '../../Context/Command.context';
+
+import './Command.scss';
+
+const Command: React.FC = () => {
+  const history = useHistory();
+  const commandContext = useContext(CommandContext);
+
+  const [output, setOutput] = useState(commandContext.message);
+
+  useEffect(() => {
+    setOutput(commandContext.message);
+  }, [commandContext.message]);
+
+  const commands = () => {
+    return {
+      cd: {
+        description: 'Back to the start.',
+        usage: 'cd',
+        fn: () => {
+          commandContext.message = '';
+          setOutput('');
+          history.push('/');
+        },
+      },
+      man: {
+        description: 'Show the manual.',
+        usage: 'man install|input|model|magic|extra|api',
+        fn: (arg: string) => {
+          if (!arg) {
+            arg = 'install';
+          }
+          commandContext.message = 'RTFM: ' + arg;
+          history.push('/man/' + arg);
+        },
+      },
+      sim: {
+        description: 'Simulate realtime updates.',
+        usage: 'sim',
+        fn: () => sim(),
+      },
+    };
+  };
+
+  const sim = () => {
```

Comment:
What does this do? It is retrieving HTMLElements and changing properties - why? Were there no React Components for that available?